core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/common/DetailedStackStatus.java

```diff
@@ -25,6 +25,7 @@ public enum DetailedStackStatus {
     VALIDATING_CLOUD_STORAGE_ON_VM(Status.UPDATE_IN_PROGRESS),
     MOUNTING_DISKS(Status.UPDATE_IN_PROGRESS),
     STARTING_CLUSTER_MANAGER_SERVICES(Status.UPDATE_IN_PROGRESS),
+    STARTING_CLUSTER_SERVICES(Status.UPDATE_IN_PROGRESS),
```

Comment:
Why is this new status needed? 
Also I remember that it's safer to push enums out first then use them, but we might have a smart mapper for this.

Reply:
Yes, we have the DefaultEnumConverter for these cases.
Its needed to be able to distinguish between flow steps and run different conclusion checkers based on this.